lib/menu.rb

```diff
@@ -0,0 +1,19 @@
+require './lib/takeaway.rb'
+
+class Menu
+
+  attr_reader :dishes
```

Comment:
How could you make your menu more flexible and allow for different sets of dishes?